docs/pages/guides/using-firebase.md

```diff
@@ -18,7 +20,9 @@ If you'd like access to the full suite of native firebase tools, we recommend us
 
 First we need to setup a Firebase Account and create a new project. We will be using the JavaScript SDK provided by Firebase, so pull it into your Expo project.
 
-`expo install firebase`.
+<Terminal cmd={[  
+  '$ expo install firebase'
+]} />
```

Comment:
```suggestion
<Terminal cmd={['$ expo install firebase']} />
```

Let's stick to one line, if we only have one command and only one prop is used, just for the code consistency:
* https://cs.github.com/expo/expo?q=%3CTerminal+cmd%3D%7B%5B